test.js

```diff
@@ -1,93 +1,301 @@
-var assert = require("assert");
-var generate = require("bit-docs-generate-html/generate");
-var path = require("path");
-
-var Browser = require("zombie"),
-	connect = require("connect");
-
-
-var find = function(browser, property, callback, done){
-	var start = new Date();
-	var check = function(){
-		if(browser.window && browser.window[property]) {
-			callback(browser.window[property]);
-		} else if(new Date() - start < 2000){
-			setTimeout(check, 20);
-		} else {
-			done("failed to find "+property);
+var path = require('path'),
+	rmrf = require('rimraf'),
+	assert = require('assert'),
+	Browser = require('zombie'),
+	express = require('express'),
+	generate = require('bit-docs-generate-html/generate');
```

Comment:
You don’t have to change this[1], but I’d recommend keeping [variable declarations on separate lines](http://javascript.crockford.com/code.html#variable%20declarations). In this case, it’d be easier to scan through the list if they were formatted the same way and alphabetical.

[1] As far as I can tell, this isn’t covered in [jQuery’s style guide](https://contribute.jquery.org/style-guide/js/)… now I see why someone (@andrejewski?) was advocating for stricter guidelines.